04.class/memo.js

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env node
+
+const command = require("./command.js");
```

Comment:
ここでは一文字目を大文字にして`Command`とした方がいいと思います。
```suggestion
const Command = require("./command.js");
```

そうするとのちにnewするときに`new Command()`となって`Command`がコンストラクタ関数であることがわかりやすくなります。（Rubyとかでnewする時と同じような見た目になるということです）

Reply:
確かに1文字目を大文字にした方が、他のメソッドや変数と区別がついて分かりやすくなりそうです🙂

1文字目を大文字に修正しました。該当するコミットは[03c8e38](https://github.com/uchihiro04/js-practices/pull/4/commits/03c8e386ef141d6a9667c2f5ad609653ccf289bc)となります。
また、コメントはいただいておりませんでしたが、`command.js`1行目につきましても以下のような形で変更を行っております。こちらも該当するコミットは[03c8e38](https://github.com/uchihiro04/js-practices/pull/4/commits/03c8e386ef141d6a9667c2f5ad609653ccf289bc)となります。

```javascript
// 修正前
const memoController = require("./memoController.js");

// 修正後（MemoControllerを使用している他のコードも変更しております）
const MemoController = require("./memoController.js");
```